src/linter/root.go

```diff
@@ -1501,19 +1523,14 @@ func (d *rootWalker) enterFunction(fun *ir.FunctionStmt) bool {
 		d.Report(fun.FunctionName, LevelNotice, "complexity", "Too big function: more than %d lines", maxFunctionLines)
 	}
 
-	var hintReturnType types.Map
-	if typ, ok := d.parseTypeNode(fun.ReturnType); ok {
-		hintReturnType = typ
-	}
-
 	d.checkCommentMisspellings(fun.FunctionName, fun.Doc.Raw)
 	d.checkIdentMisspellings(fun.FunctionName)
 	for _, p := range fun.Params {
 		d.checkVarnameMisspellings(p, p.(*ir.Parameter).Variable.Name)
 	}
 	doc := d.parsePHPDoc(fun.FunctionName, fun.Doc, fun.Params)
 	d.reportPhpdocErrors(fun.FunctionName, doc.errs)
-	phpdocReturnType := doc.returnType
+	phpDocReturnType := doc.returnType
```

Comment:
I like the idea of making names consistent, but that could be done in a separate PR, with a reference to #422.
It's OK to keep these changes in this PR, but I'm almost sure there are more cases like this.
Extra diff lines make the review longer.